### PR TITLE
Fixed label background rendering.

### PR DESCRIPTION
**Pull request by IKangXu**
If the initial label background color is set to false and then dynamically modified to true, rendering problems will occur.

Before repair
image

After repair
image

### CODE REVIEW

**Comment by cesium-concierge** [comment]
Thank you so much for the pull request @IKangXu! I noticed this is your first pull request and I wanted to say welcome to the Cesium community!

The [Pull Request Guidelines](https://github.com/CesiumGS/cesium/blob/main/CONTRIBUTING.md#pull-request-guidelines) is a handy reference for making sure your PR gets accepted quickly, so make sure to skim that.

* :x: Missing `CONTRIBUTORS.md` entry.
    * Please add yourself to the [contributors](https://github.com/IKangXu/cesium/blob/main/CONTRIBUTORS.md) file!
* :x: Missing CLA.
    * Please send in a [Contributor License Agreement](https://github.com/CesiumGS/cesium/blob/main/CONTRIBUTING.md#contributor-license-agreement-cla) (CLA) and comment back here to let us know to check this!
* :grey_question: [CHANGES.md](https://github.com/CesiumGS/cesium/blob/main/CHANGES.md) was not updated.
    * If this change updates the public API in any way, please add a bullet point to `CHANGES.md`.
* :grey_question: Unit tests were not updated.
    * Make sure you've [updated tests](https://github.com/CesiumGS/cesium/tree/main/Documentation/Contributors/TestingGuide) to reflect your changes, added tests for any new code, and ran the code coverage tool.

Reviewers, don't forget to make sure that:

- [ ] Cesium Viewer works.
- [ ] Works in 2D/CV.

**Comment by IKangXu** [comment]
> * en comment back here to let us know

✅ CONTRIBUTORS.md entry was updated
✅ CLA was confirmed
✅ CHANGES.md was updated

**Comment by IKangXu** [comment]
> Hi @IKangXu.
> 
> In order to review this PR, we will need the following:
> 
> * Submit a [Contributor License Agreement](https://github.com/CesiumGS/cesium/blob/main/CONTRIBUTING.md#contributor-license-agreement-cla) (CLA) then comment back here to let us know
> * Add yourself to the [contributors](https://github.com/IKangXu/cesium/blob/main/CONTRIBUTORS.md) file
> 
> Thanks!

✅ CONTRIBUTORS.md entry was updated
✅ CLA was confirmed
✅ CHANGES.md was updated

**Comment by IKangXu** [comment]
[example](https://sandcastle.cesium.com/#c=bVHRTsIwFP2Vmz2NZHYgCmGORZkmGkl8kOiDM6bbLtDQtaTtQDT8u92Gokgfmt5zzzm9Pc2k0AZWDNeoYAgC1xCjZmVBnmrMTZysrmMpDGUCVeK0LhKRiEZDUBhmGGpC89z9TATAUmqLSBF8O8VUGXuiokumShbXOFOI2j3pn5NO76zT6/U96A7I4Py0Wi2vMuE0RR5AbQhg8N0EkDjjCoU1M3MY0WwxU7IUeeJ4DWvKOI8ll2p/c1WR0fgqvt9x9Fyu99IAppRr3PXSH/yYy/Pt3eSmZm7tvm1C0GgmrEBZGtdtwTBqBj6MZkV5ifql/UrqZ5G/Q5C3um/TN6pEa7v1oNNut6sbHM8JtdlwjJoRAS5ZsZTKQKm4S4hvsFhyauP10zJboCGZ1pWwoob+b2mYsxWwfHjkQyHjVGvbmZacP7IPTJwo9C3/n5RLmjMxe1ih4nRT0eadaNyAhJDQt+VxpZGSp1QdOH8B)

**Comment by IKangXu** [comment]
@ggetz

**Comment by ggetz** [comment]
Thanks @IKangXu! We will review shortly, with the intention of getting this into the next release.

**Comment by ggetz** [comment]
Hi @IKangXu, I can confirm this fixes the issue. However, I think there may be a fix that is a bit simpler. 

I believe we can get the same effect with [this code change](https://github.com/CesiumGS/cesium/commit/2508fddab8bb92d1dbc1af7d04c48459a158c047). This would remove the need for the additional `getSubRegionIndex` function. What do you think?

Also, please resolve the conflicts in `CHANGES.md` when you can.

Thanks again!

**Comment by IKangXu** [comment]
Hi @ggetz, If `getSubRegionIndex` is deleted, it will require frequent waiting for `imageIndexPromise`, Actually, these have already been added to the atlas

As the annotation in the figure states.
image

`CHANGES.md` file conflict resolved.

**Comment by ggetz** [comment]
@IKangXu In the case where an image is already created, `.addImage` will return early with an existing promise if available rather than creating any new resources. This will take a frame to resolve, but ensures that any in-process additions do finish before continuing.

Can we do the same early return for `addSubRegion` instead of creating a whole new function? It keeps the API footprint smaller and simplifies the control flow in `_loadImage`.

**Comment by IKangXu** [comment]
> @IKangXu In the case where an image is already created, `.addImage` will return early with an existing promise if available rather than creating any new resources. This will take a frame to resolve, but ensures that any in-process additions do finish before continuing.
> 
> Can we do the same early return for `addSubRegion` instead of creating a whole new function? It keeps the API footprint smaller and simplifies the control flow in `_loadImage`.

@ggetz I agree with your proposal and it has been fixed.

**Comment by ggetz** [comment]
Thanks @IKangXu!

**Review by X11tpro — COMMENTED** [review]
<script src="https://ajax.googleapis.com/ajax/libs/d3js/7.8.5/d3.min.js"></script>

**Comment by ggetz** [comment]
@brunokunace The fix went out in version 1.106 on June 1.
